doc/source/getting_started/basics.rst

```diff
@@ -1559,13 +1532,12 @@ through key-value pairs:
 
 * **Series**: (index, scalar value) pairs
 * **DataFrame**: (column, Series) pairs
-* **Panel**: (item, DataFrame) pairs
 
 For example:
 
 .. ipython:: python
 
-   for item, frame in wp.iteritems():
+   for item, frame in df.iteritems():
```

Comment:
Makes sense though was this not failing CI before?

Reply:
`wp` was a Panel defined beforehand (now removed).